### PR TITLE
Add Dynamic as a signer guide

### PR DESCRIPTION
**Pull request by matthew1809**

### CODE REVIEW

File: docs/Account/signers/dynamic.md [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+  {/* Your app's components */}
+</DynamicContextProvider>;
+```
+
+:::tip
+We are using Ethers here instead of the default Viem, as Biconomy requires an Ethers signer object. Learn more about switching between Viem and Ethers [here](https://docs.dynamic.xyz/react-sdk/viem-ethers).
+
+:::tip
+To enable embedded wallets for your users, toggle it on along with email/social auth [in the dashboard](https://app.dynamic.xyz/dashboard/configurations#emailsocialsignin).
+:::
```
Comment (AmanRaj1608) [inline docs/Account/signers/dynamic.md:50]: Need to break the first tip, this is creating a nested tip
image

File: yarn.lock
Comment (AmanRaj1608) [inline yarn.lock:1]: Why `yarn.lock` change is needed?

**Review by AmanRaj1608 — COMMENTED** [review]
Overall LGTM, can simplify the sub numbering maybe. cc: @livingrockrises @himanshugarg06

File: yarn.lock
Comment (matthew1809) [reply yarn.lock:1]: reverted any changes, not sure why it did

**Review by AmanRaj1608 — APPROVED** [review]
LGTM!

File: docs/Account/signers/dynamic.md [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+// Initialize your paymaster
+const paymaster: IPaymaster = new BiconomyPaymaster({
+    paymasterUrl: 'your-paymaster-url-from-the-biconomy-dashboard',
+});
+
+```
+
+### 4b. Initialize a ECDSAOwnershipValidationModule
+
+Now you're ready to allow the user to authorize actions from their Biconomy smart account by signing messages with their Dynamic wallet. This is done by initializing a `ECDSAOwnershipValidationModule` for the user's smart account and passing in a signer from the users wallet.
```
Comment (NunoFilipeSantos) [inline docs/Account/signers/dynamic.md:108]:
```suggestion
Now you're ready to allow the user to authorize actions from their Biconomy smart account by signing messages with their Dynamic wallet. This is done by initializing a `ECDSAOwnershipValidationModule` for the user's smart account and passing in a signer from the user's wallet.
```

File: docs/Account/signers/dynamic.md [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+);
+```
+
+## 3. Configure your Biconomy bundler and paymaster
+
+Go to the [Biconomy Dashboard](https://dashboard.biconomy.io/) and configure a Paymaster and a Bundler for your app. Make sure these correspond to the desired network for your user's smart accounts. You can learn more about the dashboard [here](/category/biconomy-dashboard)
+
+## 4. Configure Biconomy to create the smart accounts
+
+Once a user signs up through Dynamic, you can create a Biconomy smart account and use their Dynamic wallet as a signer for that new smart account. This means that the user will be able to user Biconomy's account abstraction features out of the box! Just before you do that though, you need to follow two more steps:
```
Comment (NunoFilipeSantos) [inline docs/Account/signers/dynamic.md:77]:
```suggestion
Once a user signs up through Dynamic, you can create a Biconomy smart account and use their Dynamic wallet as a signer for that new smart account. This means that the user will be able to use Biconomy's account abstraction features out of the box! Just before you do that though, you need to follow two more steps:
```

**Review by NunoFilipeSantos — CHANGES_REQUESTED** [review]
2 typos left to fix.

**Comment by matthew1809** [comment]
> 2 typos left to fix.

Fixed @NunoFilipeSantos - thank you

**Review by NunoFilipeSantos — APPROVED** [review]
Thank you for fixing it. 😃

**Pull request merged**